addons/pos_restaurant/models/pos_order.py

```diff
@@ -16,6 +16,7 @@ class PosOrder(models.Model):
 
     table_id = fields.Many2one('restaurant.table', string='Table', help='The table where this order was served', index='btree_not_null', readonly=True)
     customer_count = fields.Integer(string='Guests', help='The amount of customers that have been served by this order.', readonly=True)
+    take_away = fields.Boolean(string="Take Away", default=False)
```

Comment:
Be consistent.
```suggestion
    takeaway = fields.Boolean(string="Takeaway", default=False)
```
(or modify the `pos.config` field).